mantidproject/mantid
# Disable ResNorm option on Quasi when using vanadium reduction

Fixes #13105.

To test:
- Open Indirect > Bayes > Quasi
- Load a vanadium reduction as the Resolution (e.g. `irs26173_graphite002_red.nxs`)
  - The _Use ResNorm_ option should be disabled and the option unchecked if it was checked
- Load an instrument resolution as the Resolution (e.g. `irs26173_graphite002_res.nxs`)
  - The _Use ResNorm_ option should be enabled (but not ticked)

Release notes updates [here](http://www.mantidproject.org/index.php?title=Release_Notes_3_5_Indirect_Inelastic&diff=24441&oldid=24432).
